cmd/ingress-operator/start.go

```diff
@@ -120,6 +119,7 @@ func start(opts *StartOptions) error {
 		Namespace:              opts.OperatorNamespace,
 		IngressControllerImage: opts.IngressControllerImage,
 		CanaryImage:            opts.CanaryImage,
+		CanaryCommand:          operatorcontroller.CanaryHealthcheckCommand,
```

Comment:
If we are going to hard-code the canary container command within the operator than I dont think `CanaryCommand` should be a part of this struct (or passed in to the operator via manifests).